src/connectors.js

```diff
@@ -10,20 +10,16 @@ import {
 } from '@web3-react/frame-connector'
 import {
   InjectedConnector,
-  // NoEthereumProviderError as InjectedNoEthereumProviderError,
   UserRejectedRequestError as InjectedUserRejectedRequestError,
 } from '@web3-react/injected-connector'
 import { PortisConnector } from '@web3-react/portis-connector'
-import { SquarelinkConnector } from '@web3-react/squarelink-connector'
```

Comment:
I wonder if we should include a file with some notes on which connectors are troublesome and why?

Reply:
I definitely didn't think of this in my review due to [this note](https://github.com/aragon/use-wallet/blob/e190ac267ef0e093ce692368c3ea97734658a378/scripts/build.sh#L58-L84) but it may be good to add a HACKS.md file, yes.

Reply:
Closing as this dependency has been added again.